build-server/main.sh

```
@@ -10,14 +10,14 @@ ENV_VARS="${ENV}"
 GIT_REPOSITORY_URL="https://github.com/${USERNAME}/${REPO}.git"
 
 
-git clone "$GIT_REPOSITORY_URL" 
+git clone "$GIT_REPOSITORY_URL" > /dev/null 2>&1
 cd "$REPO"
-git checkout "$BRANCH"
+git checkout "$BRANCH" > /dev/null 2>&1
 
 if [ -n "$ENV_VARS" ]; then
   echo "$ENV_VARS" | jq -r '.[] | "\(.key)=\(.value)"' > ".env"
   export $(grep -v '^#' ".env" | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)